src/components/backend-ai-session-list.ts

```diff
@@ -1120,8 +1120,8 @@ export default class BackendAiSessionList extends BackendAIPage {
           menu.appendChild(mountedFolderItem);
         }
       });
+      document.body.appendChild(menu);
```

Comment:
In this case, created menu object will be orphan object if it is not attached to body.